src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -1,3 +1,4 @@
+{% extends "layout.njk" %}
```

Comment:
this shouldn't be needed as all of these macros are called into other files that already extend layout